### PR TITLE
(1397) Show subnavigation

### PR DESCRIPTION
**Pull request by pezholio**
This adds subnavigation to the application using the MoJ [Primary Navigation pattern](https://design-patterns.service.justice.gov.uk/components/primary-navigation/), as well as updating the dashboard to show all the sections of the app - this may need some copy input from Selina, but can go in as is for now

## Screenshots

image

image

### CODE REVIEW

**Comment by pezholio** [comment]
[Subnavigation](https://trello.com/c/Ml3mmpbG/1397-subnavigation)

**Comment by richpjames** [comment]
I think theres something going on with the rebase in this one?

**Comment by pezholio** [comment]
Woah!

File: server/utils/navigationItems.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { UserDetails } from '@approved-premises/ui'
+
+import { sectionsForUser } from './userUtils'
+
+export const navigationItems = (user: UserDetails, currentPath: string) => {
+  const sections = sectionsForUser(user)
+
+  const items = sections.map(section => {
```
Comment (richpjames) [inline server/utils/navigationItems.ts:8]: Up to you but i'd find 
```
  const items = [{
    text: 'Home',
    href: '/',
    active: false,
  }, ...sections.map(section => {
  return {
      text: section.shortTitle,
      href: section.href,
      active: currentPath.startsWith(section.href),
    }
  })] 
  ``` 
  easier to read

File: server/utils/nunjucksSetup.ts [hunk 2]
```diff
@@ -165,6 +171,7 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
 
   njkEnv.addGlobal('summaryListSections', summaryListSections)
   njkEnv.addGlobal('dashboardTableRows', dashboardTableRows)
+  njkEnv.addGlobal('navigationItems', navigationItems)
```
Comment (richpjames) [inline server/utils/nunjucksSetup.ts:174]: I think this might be in the wrong commit?

**Pull request merged**